schema_utils.py

```diff
@@ -387,3 +603,88 @@ def is_valid_email(obj):
             bool. Whether the given object is a valid email.
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
+
+    @staticmethod
+    def is_valid_asciimath_expression(obj, algebraic=True):
+        """Checks if the given  obj(a string) represents a valid algebraic or
```

Comment:
Drop one of the spaces before "obj". Add a space after "obj".

Reply:
Done.